packages/gafl-webapp-service/src/pages/terms-and-conditions/__tests__/route.spec.js

```diff
@@ -63,5 +80,63 @@ describe('terms-and-conditions > route', () => {
       )
       expect(data.paymentRequired).toBe(paymentRequired)
     })
+
+    it('test licenceTypeDisplay is called with permission and mssgs', async () => {
```

Comment:
This test doesn't test licenceTypeDisplay is called with permission and msgs. It's testing that if `CONTACT_SUMMARY.page` is set to `false`, then a redirect error is thrown to `CONTACT_SUMMARY.uri`